test/lib.bash

```diff
@@ -185,20 +185,20 @@ function downstream_kitchensink_e2e_tests {
 function run_rolling_upgrade_tests {
   logger.info "Running rolling upgrade tests"
 
-  local base serving_image_version eventing_image_version eventing_kafka_image_version eventing_kafka_broker_image_version image_template channels common_opts
+  local base serving_image_version eventing_image_version eventing_kafka_image_version eventing_kafka_broker_test_images_version image_template channels common_opts
 
   serving_image_version=$(versions.major_minor "${KNATIVE_SERVING_VERSION}")
   eventing_image_version=$(versions.major_minor "${KNATIVE_EVENTING_VERSION}")
   eventing_kafka_image_version=$(versions.major_minor "${KNATIVE_EVENTING_KAFKA_VERSION}")
-  eventing_kafka_broker_image_version=$(versions.major_minor "${KNATIVE_EVENTING_KAFKA_BROKER_VERSION}")
+  eventing_kafka_broker_test_images_version=$(versions.major_minor "${KNATIVE_EVENTING_KAFKA_BROKER_TEST_IMAGES_VERSION}")
 
   # mapping based on https://github.com/openshift/release/blob/master/core-services/image-mirroring/knative/mapping_knative_v1_2_quay
   base="quay.io/openshift-knative/"
   image_template=$(
     cat <<-EOF
 $base{{- with .Name }}
 {{- if eq .      "wathola-kafka-sender"}}{{.}}:v$eventing_kafka_image_version
-{{- else if eq . "kafka-consumer"      }}knative-eventing-kafka-broker-test-kafka-consumer:knative-v$eventing_kafka_broker_image_version
+{{- else if eq . "kafka-consumer"      }}knative-eventing-kafka-broker-test-kafka-consumer:knative-v$eventing_kafka_broker_test_images_version
```

Comment:
What's the output of this if it is a commit?

This image wouldn't resolve properly:
```
knative-eventing-kafka-broker-test-kafka-consumer:knative-v$eventing_kafka_broker_test_images_version
```